tests/ansible_collections/roles/reboot/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+- include: reboot.yml
```

Comment:
Is this file necessary? Could we use `tests/ansible_collections/roles/reboot/reboot.yml` directly instead?

Reply:
Yes no problem, we can add everything to just one file.

Reply:
Changed.